modules/serverless-create-trigger-odc.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * /serverless/event_workflows/serverless-using-brokers.adoc
+
+[id="serverless-create-trigger-odc_{context}"]
+= Creating a trigger using the Developer perspective
+
+After you have created a broker, you can create a trigger in the web console *Developer* perspective.
+
+.Prerequisites
+To create triggers using the *Developer* perspective ensure that:
```

Comment:
Is this sentence necessary? I don't think we typically use lead-in sentences under Prereq sections.